### PR TITLE
Assignment 3 Completed

### PR DESCRIPTION
**Pull request by deepgandhecha7**
Screenshot (2)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    
+  </style>
+
+  
+</head>
+<body>
+<table>
+    <tr>
+        <td>
+            <img src="https://i.pinimg.com/736x/4f/88/32/4f8832629be791976d4407576798050f.jpg" width="100px">
```
Comment (Abduliitr) [inline index.html]: add alt attribute in the img tag.

**Review by Abduliitr — CHANGES_REQUESTED** [review]

**Comment by Abduliitr** [comment]
Correct (1 change requested)!

File: index.html (hunk 1 above)
Comment (deepgandhecha7) [reply index.html]: Okay will do it

**Review by Abduliitr — APPROVED** [review]

**Comment by jdsamar** [comment]
Code looks correct but the output seems not properly aligned

**Pull request merged**